docs/cudf/source/basics/nested.rst

```diff
@@ -0,0 +1,35 @@
+Nested
+======
+
+
+Struct and List datatypes
```

Comment:
We can remove this subtitle if we change the title to `Struct and List datatypes`.